autoload/phpactor/quickfix.vim

```diff
@@ -1,45 +1,15 @@
-function! phpactor#quickfix#strategy() abort
```

Comment:
Removed auto-detection - I would prefer everybody had the same base experience - and right now the native capability is more stable.